### DISCUSSION TITLE
Feature Request: Option to limit output

### DISCUSSION THREAD

**Discussion by TuxedoCatGames** [Kalanyr/gogrepo#72] [Ideas]
Just started using this awesome utility. I have it set up as a weekly task on my NAS to update my GoG manifest and download any updates. I have the task email me once complete, but the email is huge with the console output showing % complete every second. Was thinking of a "Verbose" flag, which when set to FALSE, could limit the output to just show start/end/error info for each file.

**Comment by Kalanyr** [comment]
Seems reasonable.  As a stopgap you can turn logging on, then pipe standard
out / standard error to null

**Reply by TuxedoCatGames** [reply]
I ended up adding a "noprogress" flag to the download command that just skips the progress output, but adds a "starting download 'downloading_path'" message instead.  Thank you for your work on this. I was planning on making an app with a UI to do the same thing, but this is even better as a "set it and forget it" task.
